src/lib/Libattr/master_resc_def_all.xml

```diff
@@ -1008,6 +1009,62 @@
 	<member_at_entlim>PBS_ENTLIM_NOLIMIT</member_at_entlim>
 	<member_at_struct>(struct resource_def *)0</member_at_struct>
    </attributes>
+   <attributes flag = 'SVR'>
```

Comment:
Try to decouple the feature. If we make the resource as custom resource then entire feature can be decoupled from PBSCore  .

Reply:
Decoupling would make the feature very much dependent on Admin. Also we have to consider cray alps which uses few of these attributes to do the reservation.